espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MobileSwipe.kt

```diff
@@ -68,4 +76,24 @@ class MobileSwipe : RequestHandler<MobileSwipeParams, Void?> {
 
         return null
     }
+
+    override fun handleCompose(params: MobileSwipeParams): Void? {
+        // Get a reference to the compose node
+        val nodeInteractions = getNodeInteractionById(params.elementId)
+
+        if (params.direction != null) {
+            AndroidLogger.info("Performing swipe action with direction '${params.direction}'")
+            when (params.direction) {
+                UP -> nodeInteractions.performTouchInput{swipeUp()}
+                DOWN -> nodeInteractions.performTouchInput{swipeDown()}
+                LEFT -> nodeInteractions.performTouchInput{swipeLeft()}
+                RIGHT -> nodeInteractions.performTouchInput{swipeRight()}
+                else -> throw InvalidArgumentException("Direction cannot be ${params.direction}")
+            }
+        } else{
+            throw InvalidArgumentException("Must provide direction to swipe to :up,down,right,left")
```

Comment:
consider using [require](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html) instead

Reply:
used the require instead of line 94